zerver/openapi/zulip.yaml

```diff
@@ -18085,6 +18184,25 @@ components:
               "result": "error",
               "stream": "nonexistent_stream",
             }
+    NonExistingStreamIdError:
```

Comment:
This change is also in a separate PR, #25621, but I included it here to make it an easier rebase process if/when that gets merged.